resources/lib/services/playback/stream_continuity.py

```diff
@@ -57,6 +58,7 @@ def _initialize(self, data):
         if videoid.mediatype not in [common.VideoId.MOVIE, common.VideoId.EPISODE]:
             self.enabled = False
             return
+        self.videoid = videoid
```

Comment:
this already exists as local variable, line 57
convert the local variable to bind to the class

Reply:
Done